### ISSUE TITLE
Sveltekit sdk-js back button behavior with url strategy

### ISSUE THREAD

**Issue by fatstack** [inlang/inlang#847]
## Problem
With the url strategy, there is some behavior with the back button that can result in having a different language being in the url than what the current language is.

### Expected behavior
Language of the page should always match the url if using the url strategy.

## Steps to reproduce
- Use the url strategy
- Navigate to a page that has the language in the url e.g. example.com/en/mypage
- Click to navigate back to the homepage (or some other page) and switch language e.g. to DE
- Click the back button twice to return back to example.com/en/mypage, the language will be DE and not match the url

### Additional information (screenshots?)
sdk-js 0.6.1

**Comment by samuelstroschein** [comment]
I assigned @ivanhofer who is the lead for the sveltekit SDK. FYI, he is on vacation right now. Response time can take a few days.

**Comment by ivanhofer** [comment]
We have identified the issue and ar working on a fix. As a workaround you can add the following `/routes/[lang]/+layout.server.js` file:

```ts
export const load = ({ params }) => {
	// to retrigger load function when language changes e.g. due to a popstate event
	params.lang
})
```

If you already have that file present, just add `params.lang` somewhere in the `load` function and `SvelteKit` should update the language information if you press the back button.

**Comment by ivanhofer** [comment]
fixed in version `0.6.4`
